pyproject.toml

```
@@ -0,0 +1,160 @@
+[project]
+name = "liconic_module"
+version = "0.2.0"
+description = "Software for automatting a Liconic device"
+authors = [
+    {name = "Ryan D. Lewis", email="ryan.lewis@anl.gov"},
+    {name = "Rafael Vescovi", email="ravescovi@anl.gov"},
+    {name = "Doga Ozgulbas", email="dozgulbas@anl.gov"},
+    {name = "Abe Stroka", email="astroka@anl.gov"},
+    {name = "Kyle Hippe", email = "khippe@anl.gov"},
+    {name = "Tobias Ginsburg", email = "tginsburg@anl.gov"},
+]
+dependencies = [
+    "ad_sdl.wei>=0.5.3",
+    "pydantic>=2.4",
+    "pyyaml>=6.0.1",
+    "fastapi>=0.103.2",
+    "requests>=2.31.0",
+    "pandas>=2.2.0",
+    "uvicorn>=0.21.1",
+    "open-liconic-stx==0.1.0",
```

```suggestion
    "ad_sdl.wei>=0.5.3",
    "fastapi>=0.103.2",
    "uvicorn>=0.21.1",
    "open-liconic-stx==0.1.0",
```
Removing OT2 specific dependencies